Add tabs for software and astro (#3223)
astronomer/docs: astro/cli/astro-workspace-team-list.md

id: astro-workspace-team-list
description: List Teams in a Workspace.
hide_table_of_contents: true
sidebar_custom_props: { icon: "img/term-icon.png" }
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

:::info

The behavior and format of this command differs depending on what Astronomer product you're using. Use the following tabs to change product contexts.

:::

> List all Teams in your current Workspace as well as their level of [user permissions](https://docs.astronomer.io/astro/user-permissions) within the Workspace.

Reviewer: ```suggestion
List all Teams in your current Workspace, as well as their level of user permissions within the Workspace.
```
Not sure exactly where this link should go now, but we should watch out for using Astro links outside of the tab context.

- [`astro organization team create`](cli/astro-organization-team-create.md)
- [`astro organization team list`](cli/astro-organization-team-list.md)
- [`astro workspace switch`](cli/astro-workspace-switch.md)

</TabItem>
<TabItem value="software">

## Usage

```sh

> astro workspace team list <flags>

Reviewer: ```suggestion
astro workspace team list <options>
```

```

## Options

| Option           | Description                                      | Valid Values           |
| ---------------- | ------------------------------------------------ | ---------------------- |

> | `<workspace_id>` | The Workspace you want to associate the team to. | Any valid Workspace ID |

Reviewer: ```suggestion
| `<workspace_id>` | The Workspace you want to list Teams for. | Any valid Workspace ID |
```
